src/queries.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const node = require('./graphql').queryNode;
+
+const authoredQ = node('nodes')
+			.addChild(node('author')
+								.addChild(node('login'))
+								.addChild(node('url')))
+			.addChild(node('createdAt'));
+
+const participantsQ = authoredQ
+			.addChild(node('comments', {first: 100})
+								.addChild(node('totalCount'))
+								.addChild(node('pageInfo')
+													.addChild(node('endCursor')))
+								.addChild(authoredQ));
+
+const multiQs = [
+	node('totalCount'),
+	node('pageInfo').addChild(node('endCursor')),
+	participantsQ
+];
+
+const everything = (repoName, ownerName) =>
+			node('repository', {name: repoName, owner: ownerName})
+			.addChild(node('pullRequests', {first: 100}, multiQs))
+			.addChild(node('issues', {first: 100}, multiQs));
+
+const timeFilter = (before = new Date(), after = new Date(0)) =>
+			data => data.filter(x => after <= x.createdAt <= before);
```

Comment:
wtf is this syntax? :D Never seen `<=`

Reply:
Less than or equal. 

That line was actually wrong as well, you can't say a < b < c in javascript, though it's  not an error, it's just undefined...

Reply:
Ah. Wow, hard to parse.